### PR TITLE
[stable24] Fix attachment cleanup when file names contain parenthesis

### PR DESCRIPTION
**Pull request by eneiluj**
Manual backport of #2383

### CODE REVIEW

File: tests/.phpunit.result.cache [hunk 1]
```diff
@@ -0,0 +1 @@
+{"version":1,"defects":[],"times":{"OCA\\Text\\Tests\\TextTest::testDummy":0.001,"OCA\\Text\\Tests\\TextTest::testGetAttachmentNamesFromContent":0.002}}
```
Comment (mejo-) [inline tests/.phpunit.result.cache:1]: Is this file pushed on purpose?

File: tests/clover.xml [hunk 2]
```diff
@@ -0,0 +1,1461 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mejo-) [inline tests/clover.xml:1]: Is this file pushed on purpose? Looks like a coverage report that is probably subject to change with each code change, no?

**Comment by eneiluj** [comment]
@mejo- Nice catch, thanks. We need to add some entries to the gitignore file :grin:

**Review by mejo- — APPROVED** [review]

**Pull request merged**